lib/create-organization-api.js

```diff
@@ -192,6 +243,8 @@ export default function createOrganizationApi ({ http }) {
     getOrganizationMembership,
     getOrganizationMemberships,
     getOrganizationSpaceMembership,
-    getOrganizationSpaceMemberships
+    getOrganizationSpaceMemberships,
+    getInvitation,
```

Comment:
I think these need to be more specific-- what is it an invitation for?

Then again, `getSpaceMembershipInvitation` is maybe too long... WDYT?

Reply:
Let's also add these to the type definitions :) (I know I didn't ask for this before but I think it's a good idea)

Reply:
@phoebeschmidt thx for feedback! :) maybe I could say something like: `getOrganizationInvitation()` ? WDYT?

Reply:
Although I don't disagree that `getOrganizationInvitation` is a good name, it is in fact the only kind of invitation we have in Contentful. You have to invite people to your org but once they are members you can simply add them to teams or spaces.